plugins/3rdParty/qChaiScripting/codeEditor/codeEditor.cpp

```diff
@@ -0,0 +1,546 @@
+//##########################################################################
+//#                                                                        #
+//#                CLOUDCOMPARE PLUGIN: ChaiScriptingPlugin                #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 of the License.               #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#                     COPYRIGHT: Chris S Brown                           #
+//#                                                                        #
+//##########################################################################
+
+
+#include <QtWidgets>
+#include <QPlainTextEdit>
+
+
+#include "codeeditor.h"
```

Comment:
Linux is case sensitive so it must be `#include "codeEditor.h"`

Reply:
yeah I'm living in MSVC land :( I'll get this patched up